src/main/java/Main.java

```diff
@@ -0,0 +1,172 @@
+            String name = session.attribute(SESSION_USER);
+            User user = users.get(name);
+
+//            List<Bottle> bottlePages = new ArrayList<>();
+//            String pageString = request.queryParams("page");
+//            int page = Integer.valueOf(pageString);
+//            int itemsPerPage = 5;
+//            int newPageStart = page * itemsPerPage;
+//            String url = "/";
+//
+//            for (Bottle bottle : user.myBottles){
+//                bottlePages.add(bottle);
+//                }
+//
+//            }
+//            for (int i = newPageStart; i < (newPage + itemsPerPage); i++){
+//
+//                url += "?page=" + page;
+//            }
+
+            if(user == null){
+                m.put("allBottles", allBottles);
+                return new ModelAndView(m, "index.html");
+            }
+            else {
+                m.put("user", user);
+                m.put("allBottles", allBottles);
+                return new ModelAndView(m, "index.html");
+            }
+            //m.put("bottlePages", bottlePages);
+//            m.put("url", url);
+//            m.put("itemsPerPage", itemsPerPage);
+//            m.put("page", page);
+//            m.put("newPage", newPage);
+        }, new MustacheTemplateEngine());
+
+        Spark.get("/login", (request, response) -> {
+            Map m = new HashMap();
+            return new ModelAndView(m, "index.html");
+        }, new MustacheTemplateEngine());
+
+        Spark.post("/login", (request, response) -> {
+            String name = request.queryParams("submitName");
+            String password = request.queryParams("submitPassword");
+            User user = users.get(name);
+
+            if(user == null) {
+
+                user = new User(name, password);
+                users.put(name, user);
+                Session session = request.session();
+                session.attribute(SESSION_USER, name);
+                response.redirect("/");
+            }
+            else if (user != null && request.queryParams("submitPassword").equals(user.getPassword())){
+                Session session = request.session();
+                session.attribute(SESSION_USER, name);
+                response.redirect("/");
+            }
+            else {
+                response.redirect("/login");
+            }
+            return "";
+        });
+
+        Spark.get("/create-bottle", (request, response) -> {
+            Map m = new HashMap();
+            return new ModelAndView(m, "index.html");
+        }, new MustacheTemplateEngine());
+
+        Spark.post("/create-bottle", ((request, response) -> {
+            Session session = request.session();
+            String name = session.attribute(SESSION_USER);
+            User user = users.get(name);
+
+            if(user==null){
+                response.redirect("/");
+                return "";
+            }
+
+            String bottleName = request.queryParams("bottleName");
```

Comment:
Simple naming nitpick: you are referring to a bottle so you can probably drop the bottle in front of every variable.